08-read-write-plot.Rmd

```diff
@@ -668,6 +668,43 @@ writeRaster(x = single_layer, filename = "my_raster.tif",
 
 To learn more about the compression of GeoTIFF files, read Paul Ramsey's [comprehensive blog post on this topic](https://blog.cleverelephant.ca/2015/02/geotiff-compression-for-dummies.html).
 
+## Geographic metadata
+
+Geographic metadata are a cornerstone of geographic information management. 
+They are used to describe datasets, data structures and services to ensure data can be used properly. 
+They are a key driver for making data findable, accessible, interoperable and reusable, and contribute to the implementation of FAIR principles. 
+Geographic metadata are defined by the ISO/OGC standards, in particular the ISO 19115 standard and underlying schemas. 
+These standards are widely used within spatial data infrastructures, handled through metadata catalogs.
+
+Through R, the management of geographic metadata can be done with the **geometa** package that allows writing, reading and validating geographic metadata according to the ISO/OGC standard schemas. 
+It already supports various international standards for geographic metadata information, such as the ISO 19110 (Feature catalogue), ISO 19115-1 and 19115-2 (geographic metadata for vector and gridded/imagery datasets), ISO 19119 (geographic metadata for service), and ISO 19136 (Geographic Markup Language) providing methods to read, validate and write geographic metadata from R using the ISO/TS 19139 (XML) technical specification.
+
+The creation of geographic metadata is based on an object-oriented approach.
+A base metadata object is created and its fields can be completed with dedicated methods:
+
+```{r 07-read-write-plot-m5, eval=FALSE}
+library(geometa)
+# create a metadata
+md = ISOMetadata$new()
+#... fill the metadata 'md' object
+# validate metadata
+md$validate()
+# XML representation of the ISOMetadata
+xml = md$encode()
+# save metadata
+md$save("my_metadata.xml")
+# read a metadata from an XML file
+md = readISO19139("my_metadata.xml")
+```
+
+The package comes with [examples](https://github.com/eblondel/geometa/tree/master/inst/extdata/examples).
+Given the complexity and exhaustiveness the standards, and the advanced knowledge required to use them, complementary packages such as [geoflow](https://github.com/r-geoflow/geoflow) have been built to ease and automate the management of metadata based on a simplified pivot metadata model that can be represented in a tabular form tightly coupled to data flows.
```

Comment:
This is a good point, and great that the approach is extensible, and has already been extended.